layouts/shortcodes/highlight.html

```diff
@@ -1,3 +1,4 @@
-<div class="m-text-highlight m-text-highlight--{{ (.Get 0) | markdownify }}" aria-label='{{ T (printf "highlight-%s" (.Get 0)) }}'>
-    {{ .Inner }}
+<div class="m-text-highlight m-text-highlight--{{ (.Get 0) | markdownify }}">
+    <div class="m-text-highlight__roofline"> {{ T (printf "highlight-%s" (.Get 0)) }} </div>
```

Comment:
```suggestion
    <div class="m-text-highlight__roofline"> {{ T (printf "highlight.%s" (.Get 0)) }} </div>
```

Since the i18n files are yaml and they support nesting, I'd make use of it for a simpler file structure. Example with the `en.yaml`:

```
highlight:
  important: Important Information
  inofficial: Unofficial Information
  tip: Personal Tip
```